kale/utils/logger.py

```diff
@@ -38,4 +40,10 @@ def construct_logger(name, save_dir):
     gitdiff_patch = os.path.join(save_dir, file_no_ext + ".gitdiff.patch")
     os.system(f"git diff HEAD > {gitdiff_patch}")
 
+    if log_to_terminal:
```

Comment:
@Schobs  Update https://github.com/pykale/pykale/blob/main/tests/utils/test_logger.py to test this option to improve the coverage.